package.json

```diff
@@ -7,10 +7,10 @@
   "author": "EDA",
   "license": "ISC",
   "scripts": {
-    "start": "webpack && node server",
-    "dev": "node server & webpack --watch",
+    "start": "concurrently \"webpack\" \"node server\"",
+    "dev": "concurrently \"nodemon server\" \"webpack --watch\"",
     "client": "webpack-dev-server --hot --inline",
-    "server": "node server",
+    "server": "nodemon server",
```

Comment:
None of these changes appear to be related to this ticket. If you think they are required, please create a new ticket for this work :)